.github/workflows/ci.yaml

```diff
@@ -0,0 +1,100 @@
+name: ci
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - actions
```

Comment:
I think we can remove this once we're ready to merge and don't need to debug anymore, right?

Reply:
Exactly